modules/core/src/lib/deck-picker.js

```diff
@@ -250,6 +253,9 @@ export default class DeckPicker {
     mode = 'query',
     onViewportActive
   }) {
+    // picking can only hanle up to 255 items in `layers`. Drop non-pickable layers from the list.
```

Comment:
Ditto...

Use a helper function since we are doing the same thing in two places?